app/globals.css

```diff
@@ -130,3 +130,7 @@ html.dark {
   transform: translateX(0px);
   transform: translateY(0px);
 }
+
+.prose {
+  max-width: 95%;
+}
```

Comment:
apply the class directly where prose is used instead. let's not do css if it's achievable with tailwind